web/webkit/src/main/scala/net/liftweb/http/LiftRulesGuardedSetting.scala

```diff
@@ -0,0 +1,126 @@
+package net.liftweb.http
+
+import net.liftweb.util.{HasCalcDefaultValue, LiftValue}
+
+object LiftRulesGuardedSetting {
+  type StackTrace = Array[StackTraceElement]
+
+  /**
+    * Base class for all possible violations which LiftRulesGuardedSetting warns you about
+    * @param settingName
+    * @param stackTrace
+    * @param message
```

Comment:
Down to document these parameters if they seem nonobvious, but empty @params make me quake in me boots. But not from fear 😆